folding/__init__.py

```diff
@@ -9,4 +9,4 @@
     + (1 * int(version_split[2]))
 )
 
-__OPENMM_VERSION_TAG__ = "8.1.2"
+__OPENMM_VERSION_TAG__ = "8.2"
```

Comment:
is there a 8.2.0?

Reply:
We're on it, it just gets written doesn as 8.2